apps/datafeeder/src/app/presentation/components/upload-data-rules/upload-data-rules.component.html

```diff
@@ -0,0 +1,15 @@
+<div class="text-orange-500">
+  <div class="w-20 h-1 bg-orange-500 mb-4"></div>
+  <span class="font-bold" translate>datafeeder.upload.acceptedFormats</span>
+  <div
+    *ngFor="let formats of getFormattedFormats()"
+    class="w-64 flex justify-between"
+  >
+    <span>- {{ formats[0] }}</span>
```

Comment:
if more than 2 lines of format ?

Reply:
I can add input value for numeber of columns.

Reply:
no what I mean is that here you suppose that thare are 2 elements per column.
you should just go for a one dimension list and use flew or wrap to display them as a grid